tdp_lib_dag/exporter.yml

```diff
@@ -0,0 +1,37 @@
+# Copyright 2022 TOSIT.IO
+# SPDX-License-Identifier: Apache-2.0
+
+---
+- name: exporter_node_install
+  depends_on: []
+
+- name: exporter_node_config
+  noop: yes
+  depends_on:
+    - tdp-cluster_config
+
+- name: exporter_node_start
+  depends_on:
+    - exporter_node_config
+
+- name: exporter_install
+  noop: yes
+  depends_on:
+    - exporter_node_install
+
+- name: exporter_config
```

Comment:
It is the same node as here https://github.com/TOSIT-IO/tdp-collection/blob/150642f99fbff6758421d30b3444788bf0a62490/tdp_lib_dag/exporter.yml#L19

I think in the merge DAG file, this node will override the node inside `tdp-collection` and remove the dependency on `exporter_jmx_config`.

Reply:
@rpignolet done in the last commit.

Reply:
I think we should talk about it at the next meeting. Deletion does not seem to me to be the right solution because it means that the `exporter_node_*` are not in dependencies of the `exporter_*` whereas `exporter_node_*` are components of the service `exporter_*`.

Reply:
@rpignolet we can discuss no pb :)

Reply:
Fixed as decided !